arthur_bench/client/local/client.py

```diff
@@ -396,41 +396,40 @@ def get_runs_for_test_suite(
     def get_summary_statistics(
         self,
         test_suite_id: str,
-        run_id: Optional[str] = None,
+        run_ids: Optional[list[str]] = None,
         page: int = 1,
         page_size: int = DEFAULT_PAGE_SIZE,
     ) -> TestSuiteSummary:
         test_suite_name = self._get_suite_name_from_id(test_suite_id)
         if test_suite_name is None:
             raise NotFoundError(f"no test suite with id {test_suite_id}")
 
-        runs = []
-        run_id_found = False
+        suite = self.get_test_suite(test_suite_id)
+
+        runs: list[SummaryItem] = []
         run_files = glob.glob(f"{self.root_dir}/{test_suite_name}/*/run.json")
+
+        if run_ids:
+            run_id_to_file_dict = {file.split("/")[-2]: file for file in run_files}
```

Comment:
So you're going to love this by the files are named by name not id, and we have a great `run_id_to_name.json` file for maintaining that mapping. 

you can do `self._get_suite_name_from_id(id)` to get the suite names

Reply:
My bad, updated. Good catch!